utils/nano/Makefile

```diff
@@ -13,9 +13,9 @@ PKG_RELEASE:=1
 PKG_LICENSE:=GPL-3.0+
 PKG_LICENSE_FILES:=COPYING
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=@GNU/nano
-PKG_MD5SUM:=0805c5b8c75d4fde053e4b1431270f91
+PKG_MD5SUM:=f86af39514ae74e20bef3c29cd01d1090a9aca772a70e9c9f9e70c3d14b39521
```

Comment:
It's a SHA256 sum, it's allowed :-)
https://wiki.openwrt.org/doc/devel/packages#buildpackage_variables